next/utils/useWindowSize.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react'
+
+export function useWindowSize() {
```

Comment:
instead of this you can use https://usehooks-ts.com/react-hook/use-window-size , we're already using usehooks-ts elsewhere